### PR TITLE
[control-plane-manager] added a state check before changing the master node

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
Added info about check terraform state before changing the master node.

## Changelog entries
```changes
section: control-plane-manager
type: chore
summary: Add info about checking the terraform state before changing the master node.
impact_level: low
```

### CODE REVIEW

**Review by z9r5 — DISMISSED** [review]

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 1]
```diff
@@ -35,6 +35,15 @@ Adding a master node to a static or hybrid cluster has no difference from adding
...
 
+1. **In the installer container**, run the following command to check the state before working:
+
+   ```bash
+   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> \
+     --ssh-host <MASTER-NODE-0-HOST> --ssh-host <MASTER-NODE-1-HOST> --ssh-host <MASTER-NODE-2-HOST>
+   ```
+
+   The response should tell you that Terraform does not want to change anything.
+
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/docs/FAQ.md:46]: We do this check after master replicas change, cluster needs to be converged at this point.

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
Improper place

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ Adding a master node to a static or hybrid cluster has no difference from adding
      registry.deckhouse.io/deckhouse/${DH_EDITION}/install:${DH_VERSION} bash
    ```
 
+1. **In the installer container**, run the following command to check the state before working:
+
+   ```bash
+   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> \
+     --ssh-host <MASTER-NODE-0-HOST> --ssh-host <MASTER-NODE-1-HOST> --ssh-host <MASTER-NODE-2-HOST>
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/docs/FAQ.md:35]:
```suggestion
   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> --ssh-host <MASTER-NODE-0-HOST>
```

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 3]
```diff
@@ -63,6 +72,15 @@ Adding a master node to a static or hybrid cluster has no difference from adding
...
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/docs/FAQ.md:79]:
```suggestion
   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> --ssh-host <MASTER-NODE-0-HOST>
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 4]
```diff
@@ -28,6 +28,15 @@ title: "Управление control plane: FAQ"
      registry.deckhouse.io/deckhouse/${DH_EDITION}/install:${DH_VERSION} bash
    ```
 
+1. **В контейнере с инсталлятором** выполните следующую команду, чтобы проверить состояние перед началом работы:
+
+   ```bash
+   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> \
+     --ssh-host <MASTER-NODE-0-HOST> --ssh-host <MASTER-NODE-1-HOST> --ssh-host <MASTER-NODE-2-HOST>
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:35]:
```suggestion
   dhctl terraform check --ssh-agent-private-keys=/tmp/.ssh/<SSH_KEY_FILENAME> --ssh-user=<USERNAME> --ssh-host <MASTER-NODE-0-HOST>
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 5]
```diff
@@ -152,6 +153,7 @@
         Real Node `{{"{{ $labels.node_group }}/{{ $labels.name }}"}}` state is `{{`{{ $labels.status }}`}}` comparing to Terraform state.
 
         It's important to make them equal.
+        First check what will change `dhctl terraform check`.
```
Comment (RomanenkoDenys) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:156]:
```suggestion
        First, check what will change by running the `dhctl terraform check` command.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 6]
```diff
@@ -173,6 +175,7 @@
         Terraform-state-exporter can't check difference between Kubernetes cluster state and Terraform state.
 
         Probably, it occurred because Terraform-state-exporter had failed to run terraform with current state and config.
+        First check what will change `dhctl terraform check`.
```
Comment (RomanenkoDenys) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:178]:
```suggestion
        First, check what will change by running the `dhctl terraform check` command.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 7]
```diff
@@ -200,6 +203,7 @@
         Terraform-state-exporter can't check difference between Node `{{"{{ $labels.node_group }}/{{ $labels.name }}"}}` state and Terraform state.
 
         Probably, it occurred because Terraform-manager had failed to run terraform with current state and config.
+        First check what will change `dhctl terraform check`.
```
Comment (RomanenkoDenys) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:206]:
```suggestion
        First, check what will change by running the `dhctl terraform check` command.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 8]
```diff
@@ -227,5 +231,6 @@
         Terraform-state-exporter found difference between node template from cluster provider configuration and from NodeGroup `{{`{{ $labels.name }}`}}`.
         Node template is `{{`{{ $labels.status }}`}}`.
 
+        First check what will change `dhctl terraform check`.
```
Comment (RomanenkoDenys) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:234]:
```suggestion
        First, check what will change by running the `dhctl terraform check` command.
```

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 9]
```diff
@@ -200,6 +203,7 @@
         Terraform-state-exporter can't check difference between Node `{{"{{ $labels.node_group }}/{{ $labels.name }}"}}` state and Terraform state.
 
         Probably, it occurred because Terraform-manager had failed to run terraform with current state and config.
+        First, check what will change by running the `dhctl terraform check` command.
```
Comment (z9r5) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:206]:
```suggestion
        First, run the `dhctl terraform check` command to check what will change.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 10]
```diff
@@ -173,6 +175,7 @@
         Terraform-state-exporter can't check difference between Kubernetes cluster state and Terraform state.
 
         Probably, it occurred because Terraform-state-exporter had failed to run terraform with current state and config.
+        First, check what will change by running the `dhctl terraform check` command.
```
Comment (z9r5) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:178]:
```suggestion
        First, run the `dhctl terraform check` command to check what will change.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 11]
```diff
@@ -152,6 +153,7 @@
         Real Node `{{"{{ $labels.node_group }}/{{ $labels.name }}"}}` state is `{{`{{ $labels.status }}`}}` comparing to Terraform state.
 
         It's important to make them equal.
+        First, check what will change by running the `dhctl terraform check` command.
```
Comment (z9r5) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:156]:
```suggestion
        First, run the `dhctl terraform check` command to check what will change.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 12]
```diff
@@ -131,6 +131,7 @@
         Real Kubernetes cluster state is `{{`{{ $labels.status }}`}}` comparing to Terraform state.
 
         It's important to make them equal.
+        First check what will change `dhctl terraform check`.
```
Comment (z9r5) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:134]:
```suggestion
        First, run the `dhctl terraform check` command to check what will change.
```

File: modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl [hunk 13]
```diff
@@ -227,5 +231,6 @@
         Terraform-state-exporter found difference between node template from cluster provider configuration and from NodeGroup `{{`{{ $labels.name }}`}}`.
         Node template is `{{`{{ $labels.status }}`}}`.
 
+        First, check what will change by running the `dhctl terraform check` command.
```
Comment (z9r5) [inline modules/040-terraform-manager/monitoring/prometheus-rules/terraform-manager/terraform-state-exporter.tpl:234]:
```suggestion
        First, run the `dhctl terraform check` command to check what will change.
```

**Review by z9r5 — DISMISSED** [review]

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**